src/main/java/spotify/playback/data/lyrics/GeniusLyricsScraper.java

```diff
@@ -25,6 +29,10 @@ public class GeniusLyricsScraper {
   private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36";
   private static final Gson gson = new Gson();
 
+  // Testing fetching higher resolution image - Noah
+  String geniusAlbumArtUrl = "";
+  public String getHighResURL() { return geniusAlbumArtUrl; }
```

Comment:
It's technically possible that this leads to a conflict when the webapp is barely missing the proper timing as a song changes. As you fetch this info, it should add a double check that the currently playing song does indeed match this album artwork.

Oh and btw, for code-consistency with the rest of the project, please avoid one-line { } instructions :P